src/sympc/session/session_manager.py

```diff
@@ -0,0 +1,125 @@
+"""The implementation for the SessionManager.
+
+This class holds the static methods used for the Session.
+"""
+
+# stdlib
+import operator
+import secrets
+from typing import Any
+from typing import Optional
+from uuid import UUID
+from uuid import uuid4
+
+from sympc.session.session import Session
+
+
+class SessionManager:
+    """Class used to manage sessions.
+
+    Arguments:
+        uuid (Optional[UUID]): used to identify a session manager instance.
+
+    Attributes:
+        Syft Serializable Attributes
+
+        id (UID): the id to store the session
+        tags (Optional[List[str]): an optional list of strings that are tags used at search
+        description (Optional[str]): an optional string used to describe the session
+
+
+        uuid (Optional[UUID]): used to identify a session
+    """
+
+    # Those values are not used at comparison
+    NOT_COMPARE = {"id", "description", "tags"}
+
+    __slots__ = {
+        # Populated in Syft
```

Comment:
Ahh...Did not see those -- the ```SessionManager``` should not have "id" , "tags" and "description". For the moment we do not plan to send it across parties (only the ```Session``` would be sent)